mev-build-rs/src/auctioneer.rs

```diff
@@ -49,8 +51,32 @@ fn prepare_submission(
         value: payload.fees(),
     };
     let execution_payload = to_execution_payload(payload.block());
+    let payload_envelope = ExecutionPayloadEnvelopeV3::from(payload);
```

Comment:
once you rebase from `main`, should be able to grab the blob's directly with the `sidecars` method from your other PR